clang/test/Driver/hlsl-lang-targets-spirv.hlsl

```diff
@@ -2,8 +2,8 @@
 
 // Supported targets
 //
-// RUN: %clang -target dxil-unknown-shadermodel6.2-pixel %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
-// RUN: %clang -target spirv-unknown-shadermodel6.2-library %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
+// RUN: %clang -target dxil-unknown-shadermodel6.2-compute %s -S -o /dev/null 2>&1 | FileCheck --allow-empty --check-prefix=CHECK-VALID %s
+// RUN: %clang -target spirv-unknown-shadermodel6.2-compute %s -S -o /dev/null 2>&1 | FileCheck --allow-empty --check-prefix=CHECK-VALID %s
```

Comment:
Shall shadermodel version be replaced with unknown on the spirv line?
(and triple checked to refuse it when explicitly used with spirv for now?)

Reply:
Since we still have some unanswered questions about our triple requirements, I left the sys portion as-is for now and added a TODO and https://github.com/llvm/llvm-project/issues/70051 to track. If we want to require it to be `unkown` we should do that in a separate patch, but I don't think it would add value now.